Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master
     rm get_helm.sh
 
 # Add the repos containing nginx and galaxy charts
-RUN helm repo add stable https://kubernetes-charts.storage.googleapis.com/ && \
+RUN helm repo add center https://repo.chartcenter.io && \
```

Comment:
This is JFrog's helm repository: https://chartcenter.io/ which still has the `nginx-ingress` chart. Although we should update to the non-deprecated one soon (I'll make a ticket)

Reply:
https://broadworkbench.atlassian.net/browse/IA-2358